src/main/core/manager.c

```diff
@@ -28,9 +28,14 @@
 #include "main/utility/random.h"
 #include "main/utility/utility.h"
 
-#define PRELOAD_SHIM_LIB_STR "libshadow-shim.so"
+#define PRELOAD_DUMMY_LIB_STR "libshadow_dummy.so"
+#define PRELOAD_LIBC_LIB_STR "libshadow_libc.so"
 #define PRELOAD_OPENSSL_RNG_LIB_STR "libshadow_openssl_rng.so"
 
+// Allow turning off libc preloading at run-time.
+static bool _use_libc_preload = true;
+ADD_CONFIG_HANDLER(config_getUseLibcPreload, _use_libc_preload)
+
```

Comment:
I don't think that disabling libc preload is actually an important use-case, and so maybe we don't want to support that option?

Reply:
We decided that it is useful to be able to disable the libc preload lib, but we will keep it as experimental for now.